content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,140 +9,249 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
+          Google Kubernetes Engine (GKE)
+        </td>
+        <td>
+          <u><a href="https://github.com/GoogleCloudPlatform/kubeflow-distribution">Website</a></u>
+          <br>
+          <sub><u><a href="/docs/distributions/gke/">Legacy Docs</a></u></sub>
+        </td>
+        <td>
+          2022-10-17
+          <br>
+          <sup>[<a href="https://github.com/GoogleCloudPlatform/kubeflow-distribution/releases">Release History</a>]</sup>
+        </td>
       </tr>
       <tr>
         <td>Kubeflow on IBM Cloud</td>
         <td>IBM Cloud</td>
-        <td>IBM Cloud Kubernetes Service (IKS) </td>
-        <td>{{% iks/latest-version %}}</td>
-        <td><a href="/docs/distributions/ibm/">Docs</a></td>
-        <td><a href="https://github.com/IBM/manifests/tree/{{% iks/latest-branch %}}">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Kubeflow on Nutanix</td>
-        <td>Nutanix</td>
-        <td>Nutanix Karbon</td>
-        <td>{{% nutanix/latest-version %}}</td>
-        <td><a href="/docs/distributions/nutanix/">Docs</a></td>
-        <td></td>
+        <td>
+          IBM Cloud Kubernetes Service (IKS)
+        </td>
+        <td>
+          <u><a href="https://github.com/IBM/manifests/tree/{{% iks/latest-branch %}}">Website</a></u>
+          <br>
+          <sub><u><a href="/docs/distributions/ibm/">Legacy Docs</a></u></sub>
+        </td>
+        <td>
+          Unknown
+        </td>
       </tr>
       <tr>
-        <td>Kubeflow on OpenShift</td>
+        <td>Open Data Hub</td>
         <td>Red Hat</td>
-        <td>OpenShift</td>
-        <td>1.6</td>
-        <td><a href="/docs/distributions/openshift/">Docs</a></td>
-        <td><a href="https://opendatahub.io/docs/kubeflow.html">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Argoflow</td>
-        <td>Argoflow Community</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.3</td>
-        <td>N/A</td>
-        <td><a href="https://github.com/argoflow/argoflow">External Website</a></td>
+        <td>
+          OpenShift
+        </td>
+        <td>
+          <u><a href="https://opendatahub.io/docs/kubeflow.html">Website</a></u>
+          <br>
+          <sub><u><a href="/docs/distributions/openshift/">Legacy Docs</a></u></sub>
+        </td>
+        <td>
+          2022-10-26
+          <br>
+          <sup>[<a href="https://github.com/opendatahub-io/odh-manifests/releases">Release History</a>]</sup>
+        </td>
       </tr>
       <tr>
         <td>Arrikto Kubeflow as a Service</td>
-        <td>Arrikto</td>
-        <td>Fully Managed</td>
-        <td>1.4</td>
-        <td>N/A</td>
-        <td><a href="http://kubeflow.arrikto.com/">External Website</a></td>
+        <td rowspan="2">Arrikto</td>
+        <td>
+          N/A <sup>(fully managed)</sup>
+        </td>
+        <td>
+          <u><a href="https://www.arrikto.com/kubeflow-as-a-service/">Website</a></u>
+        </td>
+        <td>
+          Unknown
```

Comment:
@kimwnasptd is there something we can reference here to add the date of your latest release?

Reply:
@kimwnasptd just to clarify, we need a publicly listed blog/changelog/release-page that confirms the release date.

For example, I have used https://docs.arrikto.com/Changelog.html for the "Arrikto Enterprise Kubeflow" distribution, but "Arrikto Kubeflow as a Service" dose not seem to have an equivalent page.